data/announcements/scheduled.yaml

```diff
@@ -130,3 +130,24 @@ announcements:
   message: |
     5 days of incredible opportunities to collaborate, learn + share with the entire community!<br />
     October 24 - 28, 2022.
+
+- name: Katacoda removal
+  startTime: 2023-03-27T08:00:00
+  endTime: 2023-04-04T00:00:00
+  # CNCF stone gray on a very dark blue
+  #
+  # A pastel yellow for the banner title
+  style: >-
+    background: #181834;
+    color: rgb(229,229,229);
+  title: |
+    <span style="color: #d6d693;">Shutdown of interactive tutorials</span>
+  message: |
+    The interactive tutorials on the Kubernetes website are being shut down.
```

Comment:
The announcement text looks good.

Does it make sense to clarify the 'Kubernetes website'?
The Kubernetes project is shutting down the interactive tutorials on the Kubernetes documentation site: `https://kubernetes.io/`.

Reply:
Maybe:
```suggestion
    The interactive tutorials on this website are being shut down.
```
?